stdgo/reflect/Reflect.hx

```diff
@@ -462,17 +477,44 @@ class Type {
 	public function numMethod():GoInt {
 		switch (gt) {
 			case GT_namedType(_,_,methods,_,_,_), GT_interface(_,_,methods):
-				return methods.length; //TODO
+				return methods.length;
 			default:
 				throw new Error("reflect.NumMethod not implemented for " + toString());
 		}
 		return 0;
 	}
 
+	public function field(index:GoInt):StructField {
+		switch gt {
+			case GT_namedType(module, name, methods, fields, interfaces, type):
+				var field = fields[index.toBasic()];
+				switch field {
+					case GT_field(name, type, tag):
+						return {
+							name: name,
+							pkgPath: "",
+							type: new Type(type),
+							tag: tag,
+							offset: 0,
+							index: null,
+							anonymous: false,
```

Comment:
if `name` == "" (or possibly "_"), `anonymous` is true. 

http://golangtutorials.blogspot.com/2011/06/anonymous-fields-in-structs-like-object.html